source/Libraries/GogLibrary/GOGMetadataProvider.cs

```diff
@@ -133,5 +137,41 @@ internal GogGameMetadata DownloadGameMetadata(Game game)
 
             return metadata;
         }
+
+        internal string RemoveDescriptionPromos(string originalDescription)
+        {
+            if (originalDescription.IsNullOrEmpty())
+            {
+                return originalDescription;
+            }
+
+            // It's possible to check if a description has a promo if they contain known promo image urls
+            if (!Regex.IsMatch(originalDescription, @"<img src=""https:\/\/items.gog.com\/(promobanners|autumn|fall|summer|winter)\/", RegexOptions.IgnoreCase))

```

Comment:
I don't think this is needed anymore. You can make this check further down where you check what the first element is, by checking src attribute.

Reply:
Agreed. Change done in https://github.com/JosefNemec/PlayniteExtensions/pull/317/commits/4295495127b676f593e806c067894e3f3eec13b7